### PR TITLE
Add Polish translation of cloud-controller-manager

### PR DESCRIPTION
**Pull request by mfilocha**
This is translated content of https://github.com/kubernetes/website/blob/master/content/en/docs/reference/glossary/cloud-controller-manager.md
to Polish.

### CODE REVIEW

File: content/pl/docs/reference/glossary/cloud-controller-manager.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+short_description: >
+  Element warstwy sterowania, który integruje Kubernetesa z zewnętrznymi usługami chmurowymi.
+aka: 
+tags:
+- core-object
+- architecture
+- operation
+---
+Element składowy {{< glossary_tooltip text="warstwy sterowania" term_id="control-plane" >}} Kubernetesa,
+który zawiera logikę sterowania specyficzną dla usług chmurowych. Cloud controller manager umożliwia
```
Comment (kpucynski) [inline content/pl/docs/reference/glossary/cloud-controller-manager.md:15]:
```suggestion
który zawiera logikę odpowiedzialną za zarządzanie dedykowanymi usługami chmur obliczeniowych. Cloud controller manager umożliwia
```

File: content/pl/docs/reference/glossary/cloud-controller-manager.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+połączenie Twojego klastra z API operatora usług chmurowych i rozdziela składniki operujące na platformie
+chmurowej od tych, które sterują jedynie samym klastrem.
+
+<!--more-->
+
+Rozdzielając zarządzanie integracją w obrębie klastra pomiędzy Kubernetesa i leżącą poniżej infrastrukturę chmurową,
```
Comment (kpucynski) [inline content/pl/docs/reference/glossary/cloud-controller-manager.md:21]:
```suggestion
Dzięki rozdzieleniu logiki zarządzającej pomiędzy klaster Kubernetesa i leżącą poniżej infrastrukturę chmurową,
```

File: content/pl/docs/reference/glossary/cloud-controller-manager.md (hunk 1 above)
Comment (mfilocha) [reply content/pl/docs/reference/glossary/cloud-controller-manager.md]: This has been reworded in the next commit.

**Pull request merged**